src/Features/CSharp/Portable/EditAndContinue/SyntaxUtilities.cs

```diff
@@ -187,7 +187,13 @@ public static SyntaxNode TryGetEffectiveGetterBody(ArrowExpressionClauseSyntax p
                 return propertyBody.Expression;
             }
 
-            return accessorList?.Accessors.Where(a => a.IsKind(SyntaxKind.GetAccessorDeclaration)).FirstOrDefault()?.Body;
+            var firstGetter = accessorList?.Accessors.Where(a => a.IsKind(SyntaxKind.GetAccessorDeclaration)).FirstOrDefault();
+            if (firstGetter == null)
+            {
+                return null;
+            }
+
+            return (SyntaxNode)firstGetter.Body ?? firstGetter.ExpressionBody?.Expression;
```

Comment:
It's possible if the property is abstract, in an interface or in presence of syntax errors. I don't think we have EnC scenarios where it can happen. If we had a unit test targeting this specific helper then I'd say yes, we should have a test case for this. We don't have unit tests for all the helpers currently.